smartcontract/service/native/auth/utils_test.go

```diff
@@ -17,37 +17,13 @@
  */
 package auth
 
-import "testing"
+import (
+	"github.com/magiconair/properties/assert"
+	"testing"
+)
 
-//{"a", "b"} == {"b", "a"}
-func testEq(a, b []string) bool {
-	if a == nil && b == nil {
-		return true
-	}
-	if a == nil || b == nil {
-		return false
-	}
-
-	if len(a) != len(b) {
-		return false
-	}
-	Map := make(map[string]bool)
-	for i := range a {
-		Map[a[i]] = true
-	}
-	for _, s := range b {
-		_, ok := Map[s]
-		if !ok {
-			return false
-		}
-	}
-	return true
-}
 func TestStringSliceUniq(t *testing.T) {
 	s := []string{"foo", "foo1", "foo2", "foo", "foo1", "foo2", "foo3"}
```

Comment:
change s to `[]string{"foo3", "foo1", "foo2", "foo", "foo1", "foo2", "foo3"}`
to check both Dedup and Sort?